lib/instrumentation/aws-sdk/nr-hooks.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const instrumentations = [
+  {
+    type: 'conglomerate',
```

Comment:
These should reference:

https://github.com/newrelic/node-newrelic/blob/a9573041f44462d6ccb287f7dccf672f54b0d7b3/lib/instrumentation-descriptor.js#L57

For example: 

https://github.com/newrelic/node-newrelic/blob/a9573041f44462d6ccb287f7dccf672f54b0d7b3/lib/instrumentation/koa/nr-hooks.js#L12